docs/substrate/events.md

```diff
@@ -1,6 +1,6 @@
 ## Events
 
-Events are emitted for certain operations on the runtime. The following sections describe the events that are part of the default Substrate runtime. 
+Events are emitted for certain operations on the runtime. The following sections describe the events that are part of the default Substrate runtime.
```

Comment:
Same comments as the previous file. Revert all the changes in this file (as per the previous one) and rather adjust with the generation script. (Not of these will survive a merge as-is)